### DISCUSSION TITLE
Does seaweedfs support hot and cold separation?

### DISCUSSION THREAD

**Discussion by qiuyuyin** [seaweedfs/seaweedfs#4104] [Q&A] [answered]
For example, our team now needs to put new files on seaweedfs, and after three months, the files will be automatically transferred to Archive Access Tier for archiving (because Archive Access Tier is cheaper)

**Comment by chrislusf** [comment, marked as answer]
seaweedfs has tiered storage
https://github.com/seaweedfs/seaweedfs/wiki/Tiered-Storage

**Reply by qiuyuyin** [reply]
Thanks a lot. Follow the wiki, if we want to move the volume,we need to enter the command line.Is there a way to set up a cron job to migrate the volume, and this cron job is specified when the volume is added?

**Reply by qiuyuyin** [reply]
get it. thanks

**Comment by astrung** [comment]
@qiuyuyin @chrislusf 
Sorry but I have another question.
After moving to Archive Access Tier, will S3 API url of files will be change ?
In case of changing URL, so can get automatically new S3 url when it change ?

**Reply by chrislusf** [reply]
No, it will not be changed.

**Reply by astrung** [reply]
@chrislusf thank you for fast response. Can you have help me another question about moving file from old volumn to new volumn without changing S3 API ?
https://github.com/seaweedfs/seaweedfs/discussions/4206
